### ISSUE TITLE
Typo in requirements.txt?

### ISSUE THREAD

**Issue by Tianmaru** [emerrf/gym-wind-turbine#6]
Hi! I recently discovered this cool repository and want to use it for Reinforcement Learning.
However, I noticed that the installation with `pip install -r requirements.txt` failed.
I am not sure, if this repository is still maintained, but the solution might be to change:

```
git+git://github.com/WISDEM/AirfoilPreppy.git@02b47dffac56dc0902769c6e084f640d91b7967e#egg=airfoilprep.py
```
to

```
git+https://github.com/WISDEM/AirfoilPreppy.git@02b47dffac56dc0902769c6e084f640d91b7967e#egg=airfoilprep.py
```

in the `requirements.txt` file. I've only seen the git installation instruction with `git+https` or `git+ssh`, I am not sure about `git+git`.

**Comment by emerrf** [comment]
Thanks for your interest. It has been a while since last time I ran this code, so it probably needs some maintenance. Did you try your suggested change? I’ll try to run it next weekend. In the mean time, you can try the docker hub instances as it has a frozen copy of the dependencies. I am curious about what you want to try with it ;)

**Comment by Tianmaru** [comment]
Yeah, with the suggested change the problem was gone - before, the connection to git and therefore the installation process failed. I updated the requirements.txt and created a pull request :+1:
We will probably try to extend the gym to use OpenFAST, but it's a nice starting point!
